File: splLab/lab3/p1.cpp
#include "bits/stdc++.h"
using namespace std;

#define ll long long
#define ull unsigned long long
#define imx INT_MAX
#define imn INT_MIN

int main() {
  /*start*/
  int t;
  cin >> t;
  while (t--) {
    int n;
    cin >> n;
    int sum = 0;
    for (int i = 1; i <= n; i = i + 2) {
      if (i == n and i % 2 == 1) {
        sum += i;
        cout << i;
      } else {
        sum += i * (i + 1);
        cout << i << " X " << (i + 1);
        if (i != n - 1) cout << " + ";
      }
    }
    cout << " = " << sum << endl;
  }
}

File: splTheory/p1.dart
void main() {
int n = 3, i, j, sum? = 0;
for(i = 0; i < n; i++) {
for(j = 0; j <= i; j++) {
if(i == j) sum += i + j;
else if(i > j) sum += i + n;
else sum += n – j;
}
}
}